utils/manifests/definitions.go

```diff
@@ -30,5 +30,10 @@ type CrdFilter struct {
 	GroupFilter   JsonPath //This will specify the path to get to group name
 	VersionFilter JsonPath //This will specify the path to get to version name. [Version should have a name field]
 	SpecFilter    JsonPath //This will specify the path to get spec
-	IsJson        bool     //Set to true if input format is json
+	ItrFilter     string   //Filter is appended with "=="<some-crd-name>")]", so no need to write it completely. Just write the part before double equals
+	ItrSpecFilter string   //Filter is appended with "=="<some-crd-name>")]", so no need to write it completely. Just write the part before double equals
+	VField        string
+	GField        string
+	IsJson        bool //Set to true if input format is json
```

Comment:
Consider document like this, it is picked up by the intellisense as well.
```suggestion
        // IsJson should be set to true if the input format is json. Defaults to false
	IsJson        bool
```